pkg/test/resourcefixture/testdata/basic/run/v1beta1/runjob/jobwithiamserviceaccount/_generated_object_jobwithiamserviceaccount.golden.yaml

```diff
@@ -35,6 +36,8 @@ status:
   creator: test@google.com
   etag: abcdef123456
   lastModifier: test@google.com
+  latestCreatedExecution:
+  - {}
```

Comment:
Empty `lastestCreatedExecution` means the "RunJob" (a placeholder for containers, volumnes) is created, then we need to trigger the Cloud Run to create an "Execution" instance. Once the Execution is started, the RunJob's status is updated. The latestCreatedExecution field is then populated with information about that specific execution.

On GCP server, whe nthe RunJob is created, this latestCreatedExecution is not empty, but an initialized object without contents, which ends up as a list with an empty value in  latestCreatedExecution:
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/controller/direct/run/mapper.generated.go#L540